src/web/app/pages-instructor/instructor-home-page/__snapshots__/instructor-home-page.component.spec.ts.snap

```diff
@@ -282,475 +226,35 @@ exports[`InstructorHomePageComponent should snap with one course with one feedba
         </div>
       </div>
     </div>
-  </div><div
-    class="row mb-3"
-  >
+  </div><tm-loading-spinner>
     <div
-      class="col-4"
-    >
-      <a
-        class="btn btn-primary text-white"
-        href="/web/instructor/courses?isAddNewCourse=true"
-        routerlink="/web/instructor/courses"
-      >
-        Add New Course
-      </a>
-    </div>
-    
-    <div
-      class="col-8"
+      class="loading-container"
     >
       <div
-        class="float-right"
+        class="row spinner-border text-primary"
+        role="status"
+      />
+      <div
+        class="row text-secondary"
       >
-        <strong
-          class="d-inline"
-        >
-           Sort By: 
-        </strong>
-        <div
-          class="btn-group"
-          data-toggle="buttons"
-        >
-          <button
-            class="btn btn-light"
-          >
-            Course ID
-          </button>
-          <button
-            class="btn btn-light"
-          >
-            Course Name
-          </button>
-          <button
-            class="btn btn-light"
-            disabled=""
-          >
-            Creation Date
-          </button>
-        </div>
+        Loading...
       </div>
     </div>
-  </div><div>
-    
+  </tm-loading-spinner><tm-loading-spinner>
     <div
-      class="card"
+      class="loading-container"
     >
       <div
-        class="card-header bg-primary text-white course-tab-header"
```

Comment:
Is the deletion of the snapshot intended? Look like we do have concrete content in the container?

Reply:
Looks like you need update set `hasCoursesLoaded ` when testing. Let's check similar case in other tests also.

Reply:
In addition, we shall have test case when the courses are loading.